Misc/NEWS.d/next/Library/2021-07-02-18-17-56.bpo-44554.aBUmJo.rst

```diff
@@ -0,0 +1,5 @@
+In Pdb, refactor handling of module and script targets, now modeling each
+with a class encapsulating the behavior for each and opening up the
+opportunity to "check" for the viability of a module target prior to
+attempting to run it. Subtly tweak the internal logic to accommodate the
+refactor.
```

Comment:
This news item would be useful to anyone trying to understand a change in behaviour of pdb, so I would change it to something simpler like:

Refactor argument processing  in :func:`pdb.main` to simplify detection of errors in input loading.

Reply:
Done in e21f085196.